pkg/cmd/grafana-cli/commands/commands.go

```diff
@@ -100,7 +100,7 @@ func runPluginCommand(command func(commandLine utils.CommandLine) error) func(co
 			return err
 		}
 
-		logger.Info(color.GreenString("Please restart Grafana after installing plugins. Refer to Grafana documentation for instructions if necessary.\n\n"))
+		logger.Info(color.GreenString("If you have recently installed a plugin, please restart Grafana. Refer to Grafana documentation for instructions if necessary.\n\n"))
```

Comment:
This one has always bugged me. WDYT if we finally moved this to the end of the relevant commands (`install`, `update`, `update-all`, `uninstall`) instead?